nixos/modules/services/monitoring/loki.nix

```diff
@@ -22,6 +22,13 @@ in {
       '';
     };
 
+    package = mkOption {
+      type = types.package;
+      default = pkgs.grafana-loki;
+      defaultText = lib.literalExpression "pkgs.grafana-loki";
+      description = lib.mdDoc "Package to use.";
+    };
```

Comment:
```suggestion
    package = lib.mkPackageOptionMD pkgs "grafana-loki" { };
```

Reply:
Oh wow, I didn't know that function existed.
Thank you!